src/WebJobs.Script/Binding/TableBinding.cs

```diff
@@ -99,6 +107,15 @@ public override CustomAttributeBuilder GetCustomAttribute()
                 boundRowKey = Resolve(boundRowKey);
             }
 
+            Attribute[] additionalAttributes = null;
+            if (!string.IsNullOrEmpty(Metadata.Connection))
+            {
```

Comment:
We need to be defensive. Not all scenarios will involve our UI. I do believe UI sets it always, but we can't count on it. Also, I've seen some templates having the property empty.